data/korath/korath missions.txt

```diff
@@ -257,3 +257,53 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+				"reputation: Korath" >= -100
+			and
+				random < 10
+				"reputation: Korath" < -100
+				"reputation: Korath" >= -1000
+			and
+				random < 25
+				"reputation: Korath" < -1000
+				"reputation: Korath" >= -10000
+			and
+				random < 50
+				"reputation: Korath" < -10000
+
+			and
+				random < 10
+				"reputation: Korath (Civilian)" < 0
+				"reputation: Korath (Civilian)" >= -300
+			and
+				random < 25
+				"reputation: Korath (Civilian)" < -300
+				"reputation: Korath (Civilian)" >= -3000
+			and
+				random < 50
+				"reputation: Korath (Civilian)" < -3000
+	source
+		government Korath "Korath (Civilian)"
+		system Alcyone Almach "Alpha Hydri" Atik Durax Misam Polaris Sheratan
+	on offer
+		conversation
+			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. You had best finish your business with this ship before yours is reduced to a ball of superheated plasma."
+				accept
+	# The destination is mandatory, but its choice won't affect anything due to "deadline 1"
+	destination "Kessel Sepret"
+	npc kill
+		government "Korath"
+		personality harvests plunders unconstrained coward uninterested entering
```

Comment:
```suggestion
		personality harvests plunders unconstrained coward uninterested entering vindictive
```
they should be angry, right? just a suggestion

Reply:
No, they have arrived to clean out a system that may be filled with hostile ships. They wouldn't waste time shooting at an exploding ship.